src/cli_config.rs

```diff
@@ -30,6 +30,16 @@ impl FromStr for ThreadsConfig {
     }
 }
 
+arg_enum! {
+    /// Specifies where to write logs.
+    #[derive(Clone, Copy, PartialEq, Eq, Debug)]
+    pub enum LogType {
+        Std,
```

Comment:
why not console?

Reply:
Maybe "Stderr" is better? Because "console" can mean stdout as well: https://github.com/sebasmagri/env_logger#configuring-log-target

Reply:
sure, Stderr looks better if the output goes to stderr